### PR TITLE
Update packaging to match that of release

### PR DESCRIPTION
**Pull request by roaksoax**
debian/*: Update packaging to make it policy compliant.
buildpackage.sh: Ensure version parsing can handle versions with '~'

### CODE REVIEW

File: debian/candid.postinst [hunk 1]
```diff
@@ -1,6 +1,14 @@
-if [ -x "/etc/init.d/candid" ]; then
```
Comment (albertodonato) [inline debian/candid.postinst]: I think you can drop this file entirely?

File: debian/preinst [hunk 2]
```diff
@@ -22,11 +15,16 @@ if ! getent passwd candid >/dev/null; then
       --system \
           --disabled-login \
```
Comment (albertodonato) [inline debian/preinst]: could you please fix indentation here?

File: debian/preinst [hunk 3]
```diff
@@ -22,11 +15,16 @@ if ! getent passwd candid >/dev/null; then
...
       --ingroup candid \
-      --gecos "identity server" \
+      --gecos "Canonical Identity server" \
       --shell /bin/false \
+      --no-create-home --home /nonexistent \
       candid  >/dev/null
 fi
 
 # create log dir
 mkdir -p /var/log/identity
```
Comment (albertodonato) [inline debian/preinst]: The log dir is not used, it can be dropped

File: debian/preinst [hunk 4]
```diff
@@ -22,11 +15,16 @@ if ! getent passwd candid >/dev/null; then
...
 chown candid:candid /var/log/identity
+
+#DEBHELPER#
+
+exit 0
```
Comment (albertodonato) [inline debian/preinst]: the `exit 0` here shouldn't be needed

File: debian/rules [hunk 5]
```diff
@@ -12,30 +12,30 @@ export PATH:=$(GOPATH)/bin:$(PATH)
...
+	dh_install bin/candidsrv usr/sbin/
+	dh_install bin/candid usr/bin/
 	cp src/github.com/CanonicalLtd/candid/cmd/candidsrv/config.yaml config.yaml.sample
 	dh_install config.yaml.sample etc/candid/
-	dh_install src/github.com/CanonicalLtd/candid/static/favicon.ico usr/share/candid/static/
-	dh_install src/github.com/CanonicalLtd/candid/static/css/style.css usr/share/candid/static/css
-	dh_install src/github.com/CanonicalLtd/candid/templates/login usr/share/candid/templates/
-	dh_install src/github.com/CanonicalLtd/candid/templates/register usr/share/candid/templates/
+	dh_install src/github.com/CanonicalLtd/candid/static/ usr/share/candid/
+	dh_install src/github.com/CanonicalLtd/candid/templates/ usr/share/candid/
```
Comment (albertodonato) [inline debian/rules]: you can move all the dh_install to a `install` file, which is a bit nicer

File: debian/candid.postinst (hunk 1 above)
Comment (roaksoax) [reply debian/candid.postinst]: this is what has landed in the distro, any future changes are future changes.

File: debian/preinst (hunk 2 above)
Comment (roaksoax) [reply debian/preinst]: this is what has landed in the distro, any future changes are future changes.

File: debian/preinst (hunk 3 above)
Comment (roaksoax) [reply debian/preinst]: this is what has landed in the distro, any future changes are future changes.

File: debian/rules (hunk 5 above)
Comment (roaksoax) [reply debian/rules]: this is what has landed in the distro, any future changes are future changes.

File: debian/preinst (hunk 4 above)
Comment (roaksoax) [reply debian/preinst]: this is what has landed in the distro, any future changes are future changes.

**Pull request merged**